index.md

```diff
@@ -8,24 +8,24 @@ As also described in the [PSR Workflow Bylaw][workflow]. The Editor, or editors,
 
 ### Accepted
 
-| Num | Title                          | Editor                         |  Coordinator            | Sponsor          |
-|:---:|--------------------------------|--------------------------------|-------------------------|------------------|
-| 1   | [Basic Coding Standard][psr1]  | Paul M. Jones                  | _N/A_                   | _N/A_            |
-| 2   | [Coding Style Guide][psr2]     | Paul M. Jones                  | _N/A_                   | _N/A_            |
-| 3   | [Logger Interface][psr3]       | Jordi Boggiano                 | _N/A_                   | _N/A_            |
-| 4   | [Autoloading Standard][psr4]   | Paul M. Jones                  | Phil Sturgeon           | Larry Garfield   |
-| 6   | [Caching Interface][psr6]      | Larry Garfield                 | Paul Dragoonis          | Robert Hafner    |
-| 7   | [HTTP Message Interface][psr7] | Matthew Weier O'Phinney        | Beau Simensen           | Paul M. Jones    |
-| 11  | [Container Interface][psr11]   | Matthieu Napoli, David Négrier | Matthew Weier O'Phinney | Korvin Szanto    |
-| 13  | [Hypermedia Links][psr13]      | Larry Garfield                 | Matthew Weier O'Phinney | Marc Alexander   |
-| 16  | [Simple Cache][psr16]          | Paul Dragoonis                 | Jordi Boggiano          | Fabien Potencier |
+| Num   | Title                            | Editor                           | Coordinator               | Sponsor            |
+| :---: | -------------------------------- | -------------------------------- | ------------------------- | ------------------ |
+| 1     | [Basic Coding Standard][psr1]    | Paul M. Jones                    | _N/A_                     | _N/A_              |
+| 2     | [Coding Style Guide][psr2]       | Paul M. Jones                    | _N/A_                     | _N/A_              |
+| 3     | [Logger Interface][psr3]         | Jordi Boggiano                   | _N/A_                     | _N/A_              |
+| 4     | [Autoloading Standard][psr4]     | Paul M. Jones                    | Phil Sturgeon             | Larry Garfield     |
+| 6     | [Caching Interface][psr6]        | Larry Garfield                   | Paul Dragoonis            | Robert Hafner      |
+| 7     | [HTTP Message Interface][psr7]   | Matthew Weier O'Phinney          | Beau Simensen             | Paul M. Jones      |
+| 11    | [Container Interface][psr11]     | Matthieu Napoli, David Négrier   | Matthew Weier O'Phinney   | Korvin Szanto      |
+| 13    | [Hypermedia Links][psr13]        | Larry Garfield                   | Matthew Weier O'Phinney   | Marc Alexander     |
+| 15    | [HTTP Handlers][psr15]           | Woody Gilk                       | _N/A_                     | Matthew Weier O'Phinney |
```

Comment:
Why all the additional spaces? :(

Reply:
I have no idea. My editor did this, I think. :-(

I'll try opening it with no syntax, and reformat it.